src/tabs/presets/presets.css

```diff
@@ -125,6 +125,11 @@
     color: var(--defaultText);
 }
 
+.presets_filter_select_nonempty {
```

Comment:
any reason why presets is the only file to use snake_case? Would be nice to agree on something a bit more consistent since we have literally every casing option possible when it comes to css 😄

Reply:
Not the only file for sure, for example main.css itself :)
But I agree we need to define CSS/HTML coding style 100%